build.gradle

```diff
@@ -63,12 +83,7 @@ dependencies {
 	runtime "com.yammer.metrics:metrics-core:$metricsVersion"
 	runtime "com.yammer.metrics:metrics-annotation:$metricsVersion"
 
-	compile("org.apache.kafka:kafka_$scalaVersion:$kafkaVersion") {
-		exclude module: 'jms'
-		exclude module: 'jmxtools'
-		exclude module: 'jmxri'
-	}
-
+	compile "org.apache.kafka:kafka_$scalaVersion:$kafkaVersion"
```

Comment:
Sorry, I mentioned this in the PR description but neglected to include it in the commit message. Here's the reason:

> Three exclusions have been removed from the Kafka dependency. The Spring IO plugin identified them as being incomplete (no group is specified so they won't appear in the project's generated pom) and, upon further investigation, it appears that they weren't actually excluding anything so they have been removed.